invoiceitem.go

```diff
@@ -33,23 +33,23 @@ type InvoiceItem struct {
 	Customer     *Customer         `json:"customer"`
 	Date         int64             `json:"date"`
 	Deleted      bool              `json:"deleted"`
-	Desc         string            `json:"description"`
+	Description  string            `json:"description"`
 	Discountable bool              `json:"discountable"`
 	ID           string            `json:"id"`
 	Invoice      *Invoice          `json:"invoice"`
-	Live         bool              `json:"livemode"`
-	Meta         map[string]string `json:"metadata"`
+	Livemode     bool              `json:"livemode"`
+	Metadata     map[string]string `json:"metadata"`
 	Period       *Period           `json:"period"`
 	Plan         *Plan             `json:"plan"`
 	Proration    bool              `json:"proration"`
 	Quantity     int64             `json:"quantity"`
-	Sub          string            `json:"subscription"`
+	Subscription string            `json:"subscription"`
```

Comment:
Since we're about to break it anyway, should we make `Subscription` expandable here?

Reply:
fixed!